### PR TITLE
Introduce `tslint` to lint for TypeScript

### PR DESCRIPTION
**Pull request by romandev**
Currently, we are using `clang-format --diff` command to lint for TypeScript.
It's useful but not tightly linter. So, we should introduce `tslint` to lint for
TypeScript tightly.

This change applies the linter to gulpfile.ts once. And then we will apply the
linter to other files in the future.

ISSUE=#205

### CODE REVIEW

File: gulpfile.ts [hunk 1]
```diff
@@ -15,70 +15,126 @@
  */
 
 import * as gulp from 'gulp';
+import * as shell from 'gulp-shell';
```
Comment (yjaeseok) [inline gulpfile.ts:18]: I think it seems to be arranged by alphabet.

**Comment by yjaeseok** [comment]
i left minor comments

**Review by romandev — COMMENTED** [review]
@yjaeseok Where is your comments?
Did you publish your comments?

File: gulpfile.ts (hunk 1 above)
Comment (romandev) [reply gulpfile.ts:18]: This is already sorted in alphabetical order.
The order is based on original package name.

Comment (yjaeseok) [reply gulpfile.ts:18]: ah ha ~ i am sorry

File: bootstrap/command/lint [hunk 2]
```diff
@@ -1,19 +0,0 @@
...
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-. $BACARDI_PATH/bootstrap/common/path_info.sh
-
-$(bootstrap_command_path)/format --diff
```
Comment (yjaeseok) [inline bootstrap/command/lint:19]: Can I understand that you deleted the lint command
to replace lint command with tslint?

Comment (romandev) [reply bootstrap/command/lint:19]: Yeah, this patch disables clang-format for TypeScript.
Instead introduce tslint for TypeScript.

File: gulpfile.ts (hunk 1 above)
Comment (romandev) [reply gulpfile.ts:18]: NP :)

**Review by yjaeseok — APPROVED** [review]
lgtm

**Pull request merged**